src/app/shared/shared.module.ts

```diff
@@ -0,0 +1,16 @@
+// Add shared components, directives, and pipes to this module.
+// Do not add global services to this module, add to CoreModule instead.
+
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+@NgModule({
+  imports: [
+    CommonModule,
+  ],
+  declarations: [],
+  exports: [
+    CommonModule,
```

Comment:
Might be my unfamiliarity with ngmodule - but do we need to re-export `@angular/common` from our own modules?